apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -28,8 +31,12 @@ function SectionProgressSelector({
   }
 
   const toggleV1OrV2Button = () => (
-    <div>
-      <Button onClick={onShowProgressTableV2Change}>Toggle V1/V2</Button>
+    <div style={{float: 'right'}}>
```

Comment:
Let's use `module.scss` instead of style. Also, what do you think about adding some margin to the right? The mocks look less glued to the side to me

Reply:
Do we need the div or can we just use the <Link>?

Reply:
Added the margin but struggled to remove the div - it caused a lot of other problems on the page, so I kept it in there for now.